src/main/kotlin/org/kamiblue/client/module/modules/render/NewChunks.kt

```diff
@@ -116,4 +161,25 @@ internal object NewChunks : Module(
             }
         }
     }
+
+    // p2.x > p1.x and p2.y > p1.y is assumed
+    private fun isSquareInRadius(p1: Vec2d, p2: Vec2d, radius: Float): Boolean {
+        return if ((p1.x + p2.x) / 2 > 0) {
+            if ((p1.y + p2.y) / 2 > 0) {
+                p2.length()
+            } else {
+                Vec2d(p2.x, p1.y).length()
+            }
+        } else {
+            if ((p1.y + p2.y) / 2 > 0) {
+                Vec2d(p1.x, p2.y).length()
+            } else {
+                p1.length()
+            }
+        } < radius
```

Comment:
I'll try to find some way to rewrite this to be easier to read

Reply:
In the last commit I also made this a bit easier to read, but feel free to change it further.